src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
That doesn't work either. You report not the site, but the PNN. And that is T0_CH_CERN_MSS, which is not the pnn for T0_CH_CERN. The real fix here is to break the association between PNN and site in the WMBS schema, but that is something that has potential consequences for all of resource control and how we do location handling in WMCore.